webapplication/aoi/management/commands/notebook_executor.py

```diff
@@ -4,8 +4,8 @@
 from aoi.management.commands._notebook import (
     NotebookDockerThread, 
     PublisherThread, 
-    NotebookK8sThread
 )
+from notificator.management.commands.notification_thread import NotificationThread
```

Comment:
aoi in this way depends on notificator

Reply:
I used another approach with save imports